features/shared/entering_parent_branch.feature

```diff
@@ -31,6 +31,7 @@ Feature: Entering a parent branch name when prompted
     When I run `git sync` and enter "2" and "1"
     Then I see "Please specify the parent branch of feature-2"
     And I see "Please specify the parent branch of feature-1"
+    And I only see the branch list once
```

Comment:
It seems odd that this would be the only one to do that though. The real thing I wanted is to only see the branch list and explanation once. This verifies the actual branch list and explanation which could be something every test should do.